packages/react-app/src/services/auth.service.ts

```diff
@@ -93,6 +80,27 @@ export const createCache = async ({
 	return newCache;
 };
 
+export const getPermissions = async (accessToken: string, customerId: string): Promise<Role[]> => {
+	/**
+   * Returns a list of permissions for the current user
+   * We currently only support the `bounty-create` role, for discord level 1 - 4s in bankless DAO
+   */
+	// note: only currently supports bankless
+	const banklessRolesForUser = await getRolesForUserInGuild(accessToken, customerId);
+	const permissions: Role[] = [];
+	if (
+		banklessRolesForUser.includes(BANKLESS_ROLES.LEVEL_4) ||
+		banklessRolesForUser.includes(BANKLESS_ROLES.LEVEL_3) ||
+		banklessRolesForUser.includes(BANKLESS_ROLES.LEVEL_2) ||
+		banklessRolesForUser.includes(BANKLESS_ROLES.LEVEL_1)
```

Comment:
When we move to multitenant, this will just be a lookup from the applicable roles or something similar